### ISSUE TITLE
keyboard shortcuts not working without toolbar

### ISSUE THREAD

**Issue by miko007** [sparksuite/simplemde-markdown-editor#161]
some keybindings in simplemde seem to be bound to the toolbar. if one specifies an instance like so:

``` javascript
$(document).ready(function() {
    simplemde = new SimpleMDE({
        element: $("#editor")[0],
        toolbar: false
    });
});
```

one would no longer be able to i.e. toggle preview with CTRL+P
(neither in firefox nor in chromium)

**Comment by WesCossick** [comment]
I'll look into improving this design when SimpleMDE undergoes a rewrite for v2.0.0.

**Comment by WesCossick** [comment]
Fixed with PR #238
